tests/acceptance/AuthorshipCest.php

```diff
@@ -0,0 +1,135 @@
+	 * @param AcceptanceTester $I Actor object.
+	 *
+	 * @return void
+	 */
+	public function _before( AcceptanceTester $I ) {
+		$I->bootstrapWith( [ __CLASS__, '_enableAuthorship' ] );
+	}
+
+	/**
+	 * Activate authorship.
+	 *
+	 * @return void
+	 */
+	public static function _enableAuthorship() {
+		add_filter( 'altis.config', function( $config ) {
+			$config['modules']['cms']['authorship'] = true;
+			return $config;
+		} );
+	}
+
+	/**
+	 * Test creating a new post with both a guest and an author user.
+	 *
+	 * @param AcceptanceTester $I Tester
+	 *
+	 * @return void
+	 */
+	public function testAuthorship( AcceptanceTester $I ) {
+		$I->haveUserInDatabase( 'Arthur', 'author' );
+
+		$I->wantToTest( 'I can add multiple authors' );
+		$I->loginAsAdmin();
+
+		// Go to new post page.
+		$I->amOnAdminPage( 'post-new.php' );
+		$I->wait( 10 );
+
+		// Add a title.
+		$I->click( '.editor-post-title__input' );
+		$I->type( 'Test post 1' );
+
+		// Authorship input exists.
+		$I->seeElement( '.authorship-select__input' );
+
+		// Add a guest author.
+		$I->click( '.authorship-select__input input' );
+		$I->type( 'Gusto' );
+		$el = Locator::contains( 'div.authorship-select__option', 'Create "Gusto"' );
+		$I->waitForElementVisible( $el, 20 );
+		$I->click( $el );
+		$I->waitForElement( '.authorship-select__multi-value__label' );
+
+		// Add an existing author.
+		$I->click( '.authorship-select__input input' );
+		$I->type( 'Arthur' );
+		$el = Locator::contains( 'div.authorship-select__option', 'Arthur' );
+		$I->waitForElementVisible( $el, 20 );
+		$I->click( $el );
+		$I->waitForElement( '.authorship-select__multi-value__label:nth-child(1)' );
+
+		// Publish the post.
+		$I->click( '.editor-post-publish-button__button' );
+		$I->click( '.editor-post-publish-panel__header-publish-button .editor-post-publish-button__button' );
+		$el = Locator::contains( '.editor-post-publish-panel__header-publish-button .editor-post-publish-button__button', 'Publishing' );
+		$I->waitForElementNotVisible( $el, 20 );
+
+		// Check post is published correctly.
+		$I->seePostInDatabase( [
+			'post_title' => 'Test post 1',
+			'post_status' => 'publish',
+		] );
+
+		// Check both guest and author users are added.
+		$I->seeUserInDatabase( [ 'user_login' => 'Gusto' ] );
+		$post_id = $I->grabFromDatabase( $I->grabPostsTableName(), 'ID', [ 'post_title' => 'Test post 1' ] );
+		$guest_user_id = $I->grabUserIdFromDatabase( 'Gusto' );
+		$author_user_id = $I->grabUserIdFromDatabase( 'Arthur' );
+		$guest_term_id = $I->grabTermIdFromDatabase( [ 'slug' => $guest_user_id ] );
+		$author_term_id = $I->grabTermIdFromDatabase( [ 'slug' => $author_user_id ] );
+		$I->seePostWithTermInDatabase( $post_id, $guest_term_id, null, 'authorship' );
+		$I->seePostWithTermInDatabase( $post_id, $author_term_id, null, 'authorship' );
+
+		// Test post API response has expected authorship values.
+		$I->amOnPage( '/wp-json/wp/v2/posts/' . $post_id );
+		$source = $I->grabTextFrom( 'pre' );
```

Comment:
Is this perhaps browser dependent? I'm not sure if there's a uniform way browsers render JSON responses. Maybe it could be `<body>`?

Reply:
Yeah, will need to test with Firefox. Good catch!